package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cfpb-chart-builder",
-  "version": "1.2.0",
+  "version": "2.0.0",
```

Comment:
I don't know why github shows a diff here, when this is the same line in master :woman_shrugging: